### PR TITLE
Aws 20.1.0

### PR DESCRIPTION
**Pull request by whites11**
Towards: https://github.com/giantswarm/roadmap/issues/3376

### Checklist
- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### CODE REVIEW

File: aws/v20.0.1/release.yaml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    version: 3.2.1
+  - dependsOn:
+    - prometheus-operator-crd
+    name: chart-operator-extensions
+    version: 1.1.2
+  - componentVersion: 1.15.1
+    dependsOn:
+    - aws-cloud-controller-manager
+    name: cilium
+    version: 0.22.0
```
Comment (mcharriere) [inline aws/v20.0.1/release.yaml:53]: hey @whites11 I think we are good to go with this upgrade. All the tests I ran show it's fine to skip 2 versions of cilium; so no network disruption during the upgrade. (that would be the case of updating directly to this release from a v19 release)

Comment (whites11) [reply aws/v20.0.1/release.yaml:53]: ok, thanks!

File: aws/v20.1.0/announcement.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+**Workload cluster release v20.1.0 for AWS is available**. This patch release fixes an issue with clusters running in IMDSv2 mode only as well as bringing updated version of Flatcar Container Linux.
```
Comment (alex-dabija) [inline aws/v20.1.0/announcement.md:1]: I would replace `patch release` with `release`.

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**